src/entities/revision.js

```diff
@@ -6,5 +6,6 @@ export default class Revision extends Record( {
 	title: undefined,
 	user: undefined,
 	timestamp: undefined,
-	comment: undefined
+	comment: undefined,
+	commenthidden: undefined
```

Comment:
@dayllanmaza Did you want to make this `false` by default?

Reply:
Yup. changing it